estate/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+{
+    'name': "estate app",
+
+    'summary': """
+        estate module"
+    """,
+
+    'description': """
+        estate module"
+    """,
+
+    'author': "Odoo",
+    'website': "https://www.odoo.com/",
+
+    # Categories can be used to filter modules in modules listing
+    # Check https://github.com/odoo/odoo/blob/15.0/odoo/addons/base/data/ir_module_category_data.xml
+    # for the full list
+    'category': 'Tutorials/estate',
+    'version': '0.1',
+
+    # any module necessary for this one to work correctly
+    'depends': ['base', 'web'],
```

Comment:
```suggestion
    'depends': ['base'],
```
There is no need of web dependency, remove dependency of web.